server/xpub-controller/supportingFiles.js

```diff
@@ -9,13 +9,59 @@ class SupportingFiles {
     this.user = user
   }
 
+  async upload(file) {
+    const { stream, filename, mimetype: mimeType } = await file
+    let fileEntity = new FileModel({
+      manuscriptId: this.manuscriptId,
```

Comment:
`this.manuscriptId` hasn't been set in the constructor, so I think you'll have to pass is in as a parameter for now.